code/game/machinery/doors/airlock.dm

```diff
@@ -979,7 +979,7 @@ About the new airlock wires panel:
 	else if(istype(C, /obj/item/pai_cable))	// -- TLE
 		var/obj/item/pai_cable/cable = C
 		cable.plugin(src, user)
-	else if(istype(C, /obj/item/paper) || istype(C, /obj/item/photo))
+	else if((istype(C, /obj/item/paper) && !istype(C, /obj/item/paper/talisman)) || istype(C, /obj/item/photo))
```

Comment:
Gross check but it's the best it'll be till cult is reworked. (yes a var in the paper will work but that's also gross for just this)